src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php

```diff
@@ -103,6 +106,119 @@ public function testDenormalizeWithExtraAttributesAndNoGroupsWithMetadataFactory
         );
     }
 
+    public function testDenormalizeWithDuplicateNestedAttributes()
+    {
+        $this->expectException(LogicException::class);
+        $this->expectExceptionMessage('Duplicate serialized path: "one,two,three". Used for properties "foo" and "bar".');
```

Comment:
should be  `Duplicate serialized path: "one,two,three" used for properties "foo" and "bar".` I guess?